tests/vmi_multus_test.go

```diff
@@ -662,27 +658,19 @@ var _ = Describe("Multus", func() {
 				Expect(err).ToNot(HaveOccurred(), "Should create VMI successfully")
 				tests.WaitForSuccessfulVMIStart(agentVMI)
 
-				getOptions := &metav1.GetOptions{}
-				var updatedVmi *v1.VirtualMachineInstance
+				// Need to wait for cloud init to finish and start the agent inside the vmi.
+				tests.WaitAgentConnected(virtClient, agentVMI)
 
-				// Need to wait for cloud init to finnish and start the agent inside the vmi.
+				getOptions := &metav1.GetOptions{}
 				Eventually(func() bool {
```

Comment:
just a small nit, if we add the `tests.WaitAgentConnected` should we remove the eventually and just get the object?

Reply:
I thought that it may time for the agent to report the interface, even if connected. Am I wrong?

Reply:
Oh you are right!